.travis/test.sh

```diff
@@ -56,6 +59,10 @@ fi
 
 conda install numpy nose scipy scikit-learn pandas matplotlib python-graphviz pytest
 
+if [[ $TRAVIS_OS_NAME == "osx" ]] && [[ $COMPILER == "clang" ]]; then
+    ln -sf `ls -d "$(brew --cellar libomp)"/*/lib`/* $CONDA_PREFIX/lib || exit -1  # fix "OMP: Error #15: Initializing libiomp5.dylib, but found libomp.dylib already initialized." (OpenMP library conflict due to conda's MKL)
```

Comment:
should we add this to our document as a suggestion?

Reply:
@chivee This may be useful as a starting point for users, I'll add this to the new FAQ entry